OpenRA.Mods.Common/Widgets/Logic/Ingame/ReplayControlBarLogic.cs

```diff
@@ -46,17 +46,17 @@ public ReplayControlBarLogic(Widget widget, World world, OrderManager orderManag
 				var originalTimestep = world.Timestep;
 
 				var pauseButton = widget.Get<ButtonWidget>("BUTTON_PAUSE");
-				pauseButton.IsVisible = () => world.ReplayTimestep != 0 && orderManager.NetFrameNumber < replayNetTicks;
+				pauseButton.IsVisible = () => world.ReplayTimestep != 0 && !world.PauseStateLocked && orderManager.NetFrameNumber < replayNetTicks;
```

Comment:
Fwiw, just because `PauseStateLocked` is set, the world doesn't need to be paused.

Reply:
That is true, I'll make it pause also after a fatal lua crash and on mission objective fair

Reply:
I think @abcdefg30 means that the `PauseStateLocked` check here and on `playButton` should actually be on `.Disabled`?

Reply:
Yes. I think we only lock the pause state after pausing at the moment, but how should the user know that and we probably shouldn't rely on that assumption ourselves either.

Reply:
2 buttons overlap, that's why there is the IsVisible in the first place. One button becomes visible while the other is non-visible

Reply:
PauseStateLocked was removed, using IsGameOver now instead